Source/RMG/UserInterface/Dialog/AboutDialog.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Rosalie's Mupen GUI - https://github.com/Rosalie241/RMG
+ *  Copyright (C) 2020 Rosalie Wanders <rosalie@mailbox.org>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License version 3.
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+#include "AboutDialog.hpp"
+
+#include "Config.hpp"
+
+using namespace UserInterface::Dialog;
+
+AboutDialog::AboutDialog()
+{
```

Comment:
This should accept have an argument `QObject* parent` and pass it through the `QDialog` like
```cpp
AboutDialog::AboutDialog(QObject* parent) : QDialog(parent, Qt::WindowSystemMenuHint | Qt::WindowTitleHint)
{
}
```